app/src/main/java/swati4star/createpdf/model/TextToPDFOptions.java

```diff
@@ -157,6 +157,5 @@ public TextToPDFOptions build() {
                     mPassword, mInFileUri, mFontSize, mFontFamily, mFontColor, mPageColor);
         }
 
-
```

Comment:
If you are fixing whitespaces, it would be better to do it for whole repository. Changing it for just 1 file isn't that helpful. Thank you :)